content/courses/onchain-development/anchor-pdas.md

```diff
@@ -861,9 +865,10 @@ console.
 ```
 
 If you need more time with this project to feel comfortable with these concepts,
-feel free to have a look at
-the [solution code](https://github.com/Unboxed-Software/anchor-movie-review-program/tree/solution-pdas) before
-continuing.
+feel free to have a look at the
+[solution code](https://github.com/Unboxed-Software/anchor-movie-review-program/tree/solution-pdas)
```

Comment:
use https://github.com/solana-developers/anchor-movie-review-program instead of this repo, everywhere.